.github/workflows/main.yml

```diff
@@ -145,6 +145,16 @@ jobs:
         run: |
           python scripts/make_docs.py
           git diff --exit-code
+      - name: Report failures
+        if: always() && github.event_name == 'push'
```

Comment:
This should only run on `main`, in case we have `push` events to a branch for a PR. But we are only allowing this event on `main` and tags, so we are fine.